docs/nightly/en/user-guide/continuous-aggregation/define-time-window.md

```diff
@@ -0,0 +1,29 @@
+# Define Time Window
+
+Time window is an important attribute of your continuous aggregation query. It defines how the data is aggregated in the flow. GreptimeDB provides two types of time windows: `hop` and `tumble`, or "sliding window" and "fixed window" in other words. You can specify the time window in the `GROUP BY` clause using `hop()` function or `tumble()` function respectively. These two functions are only supported in continuous aggregate queries's `GROUP BY` position.
+
+# Tumble
+
+`tumble()` defines fixed windows that do not overlap. It signature is like the following:
+
+```
+tumble(col, interval, <start_time>)
+```
+
+Where `col` specifies use which column to compute the time window. The provided column must have a timestamp type. `interval` specifies the size of the window. The `tumble` function divides the time column into fixed-size windows and aggregates the data in each window.
+
+`start_time` is an optional parameter to specify the start time of the first window. If not provided, the start time will be aligned to calender.
+
+# Hop (not supported yet)
```

Comment:
If this feature is not supported in v0.8, we can comment this paragraph to remove it temporarily.

Reply:
Updated